frontend/packages/console-shared/src/components/spotlight/StaticSpotlight.tsx

```diff
@@ -19,7 +19,7 @@ const StaticSpotlight: React.FC<StaticSpotlightProps> = ({ element }) => {
     : {};
   return clientRect ? (
     <Portal>
-      <div className="pf-c-backdrop ocs-spotlight__with-backdrop">
```

Comment:
Difference between static and interactive Spotlight is because of the `__with-backdrop` class